### ISSUE TITLE
Problem with get_account_info

### ISSUE THREAD

**Issue by kiwandalabs** [Floris272/py-mexc-api#1]
Hi, I have been trying your api wrapper, and I found an error when trying to call the get_account_info function. 

It always seems to generate an error because of invalid "signature".

Any clue about this?

**Comment by Floris272** [comment]
Hi, for me the following example works:

```python
from mexc_api.spot import Spot

KEY = "YOURKEY"
SECRET = "YOURSECRET"

spot = Spot(KEY, SECRET)
print(spot.account.get_account_info())
```

Do you have "View Account Details" enabled?

And if it is still broken, could you check if it works with postman: https://github.com/mexcdevelop/mexc-api-postman

**Comment by kiwandalabs** [comment]
Yes, I had, but for any reason I was getting that problem. I deleted the API key and got a new one and now it's working. 
Easy solution at the end.  By the way, I prefer your api wrapper to the official one. Good job!
